exercises/strings/capitalize/capitalize.js

```diff
@@ -10,8 +10,11 @@
  * @returns {string} A capitalized copy of the string
  */
 
-function capitalize(num) {
-  // This is your job. :)
+function capitalize(num) { 
+  var a = 'The mother gave her daughter a cookie'
```

Comment:
Re-read the crash course section on functions. It's common for beginners to not quite get the idea of passing arguments to a function; play around with the examples in the crash course until you understand them.